onnx/onnx.proto

```diff
@@ -64,15 +65,25 @@ message AttributeProto {
   repeated GraphProto graphs = 11;    // list of graph
 }
 
+// Defines information on value, including the name, the type, and
+// the shape of the value.
+message ValueInfoProto {
```

Comment:
this would be a breaking change for PyTorch which spits out the proto in C++: https://github.com/pytorch/pytorch/blob/5deacb5bce8bfb41662b57a61ea635c44d2a347f/torch/csrc/onnx/onnx.h#L249

Just FYI for @ezyang